test/core-flags/cache-flags.test.js

```diff
@@ -81,6 +81,18 @@ describe('cache related flags from core', () => {
         expect(newRun.exitCode).toEqual(0);
     });
 
+    it('should assign cache build dependencies correctly when cache type is filesystem in config', () => {
+        const { stderr, stdout } = run(__dirname, ['-c', './webpack.cache.config.js']);
+        expect(stderr).toBeFalsy();
+        expect(stdout).toContain('buildDependencies: { config: [Array]');
```

Comment:
We need expand array here :smile: and let's add this check for other tests too

Reply:
done